### PR TITLE
Fix the article content voter

### PR DESCRIPTION
**Pull request by aschempp**
The `$subject->getCurrentPid()` of a content element returns the ID of the article. `ContaoCorePermissions::USER_CAN_EDIT_ARTICLES` needs the ID of the page though.

### CODE REVIEW

File: core-bundle/src/EventListener/DataContainer/DefaultOperationsListener.php [hunk 1]
```diff
@@ -186,7 +186,7 @@ private function accessChildrenCallback(string $ctable, string $table): \Closure
     {
         return function (DataContainerOperation $operation) use ($ctable, $table): void {
             $data = [
-                'pid' => $operation->getRecord()['pid'] ?? null,
+                'pid' => $operation->getRecord()['id'] ?? null,
```
Comment (leofeyer) [inline core-bundle/src/EventListener/DataContainer/DefaultOperationsListener.php:189]: Is this a related change?

**Review by leofeyer — APPROVED** [review]
Besides the possibly unrelated change, this works fine.

File: core-bundle/src/EventListener/DataContainer/DefaultOperationsListener.php (hunk 1 above)
Comment (aschempp) [reply core-bundle/src/EventListener/DataContainer/DefaultOperationsListener.php:189]: yes, that is why the current voter worked in the article list (children operation), but it is inconsistent with what DC_Table votes on in the parent view (tl_content).

**Comment by leofeyer** [comment]
Thank you @aschempp.
